lib/active_merchant/billing/gateways/commerce_hub.rb

```diff
@@ -112,6 +112,7 @@ def scrub(transcript)
         transcript.
           gsub(%r((Authorization: )[a-zA-Z0-9+./=]+), '\1[FILTERED]').
           gsub(%r((Api-Key: )\w+), '\1[FILTERED]').
+          gsub(%r(("apiKey\\?":\\?")\w+), '\1[FILTERED]').
```

Comment:
Interesting - is this field being echoed back to us in the response?

Reply:
yes the remote scrub test was failing because this apiKey field was not being scrubbed out!